lib/spack/docs/packaging_guide.rst

```diff
@@ -497,6 +497,28 @@ some examples:
 In general, you won't have to remember this naming convention because
 :ref:`cmd-spack-create` and :ref:`cmd-spack-edit` handle the details for you.
 
+-----------
+Maintainers
+-----------
+
+In Spack, packages can optionally have one or more maintainers. The ``maintainers``
```

Comment:
Should probably clarify what a maintainer is and is not. We do this already in the packaging guide but maybe we should copy the text here too.

Reply:
Moved the description of what a maintainer is here, and reference that description in "Create a New Package"